AlfrescoApp/Supporting Files/en.lproj/Localizable.strings

```diff
@@ -674,13 +674,13 @@
  */
 "settings.title" = "Settings";
 "settings.general.title" = "General";
-"settings.sync.on.cellular" = "Cellular Sync";
-"settings.sync.on.cellular.description" = "Switch off to only sync favorite content when you have a Wi-Fi connection.";
+"settings.sync.on.cellular" = "Synced Content";
```

Comment:
@andyhealey26 I disagree with this change, as I believe it could cause users to think the entire "Synced Content" feature is enabled or disabled with this toggle switch.